### PR TITLE
Fixes water vapor and cleans up some req defines.

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About The Pull Request
Fixes the turf check on water vapor condensation being inverted.
Re-uses the define for tritium combustion temperature.
Makes a define for the maximum BZ formation temperature. Makes it more clearly in reference to combustion temps.

## Why It's Good For The Game
We want slippery floors, not slippery pipes.
Also, the define for the max BZ formation temperature should be more clearly in reference to combustion temps when it's relative to them.

## Changelog

:cl:
fix: Water vapor should wet and freeze floors once again.
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -110,7 +112,7 @@
 /datum/gas_reaction/miaster/init_reqs()
 	requirements = list(
 		/datum/gas/miasma = MINIMUM_MOLE_COUNT,
-		"MIN_TEMP" = MIASTER_STERILIZATION_TEMP
+		"MIN_TEMP" = MIASTER_STERILIZATION_TEMP,
```
Comment (Ghilker) [inline code/modules/atmospherics/gasmixtures/reactions.dm:115]: Why add a `,` in the end of every list?

Comment (TemporalOroboros) [reply code/modules/atmospherics/gasmixtures/reactions.dm:115]: It's a code style thing that some maints insist on for lists defined over multiple lines.

Comment (Ghilker) [reply code/modules/atmospherics/gasmixtures/reactions.dm:115]: Well is not enforced, but whatever

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 2]
```diff
@@ -77,14 +77,16 @@
...
 
 /datum/gas_reaction/water_vapor/init_reqs()
-	requirements = list(/datum/gas/water_vapor = MOLES_GAS_VISIBLE)
+	requirements = list(
+		/datum/gas/water_vapor = MOLES_GAS_VISIBLE,
+		"MAX_TEMP" = WATER_VAPOR_CONDENSATION_POINT,
+	)
 
-/datum/gas_reaction/water_vapor/react(datum/gas_mixture/air, datum/holder)
+/datum/gas_reaction/water_vapor/react(datum/gas_mixture/air, turf/open/location)
```
Comment (LemonInTheDark) [inline code/modules/atmospherics/gasmixtures/reactions.dm:85]: keep the default typing, it's called holder at base, shouldn't rename it just here

**Pull request merged**